tests/draft2019-09/refRemote.json

```diff
@@ -37,6 +37,25 @@
             }
         ]
     },
+    {
+         "description": "anchor within remote ref",
+         "schema": {
+             "$schema": "https://json-schema.org/draft/2010-09/schema",
```

Comment:
This is an incorrect URI. Any tests that try to use this schema ought to be failing due to the missing resource.

Reply:
[Fixed](https://github.com/json-schema-org/JSON-Schema-Test-Suite/commit/c49ba54456c02ebfad3a3f3cf4718afe6da5cc40), thanks.

Reply:
just raised PR #644 to fix this